docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx

```diff
@@ -22,19 +47,104 @@ auth_service:
 
 The `license_file` path can be either absolute or relative to the configured
 `data_dir`. If the license file path is not set, Teleport will look for the
-`license.pem` file in the configured `data_dir`.
+`license.pem` file in the configured `data_dir`, which is `/var/lib/teleport` by
+default.
+
+</TabItem>
+<TabItem label="Kubernetes">
+
+Rename your license file to `license.pem`.
+
+Create a secret called "license" in the namespace you are using to deploy
+Teleport:
+
+```code
+$ kubectl -n teleport create secret generic license --from-file=license.pem
+```
+
+For the Teleport pod to load your license:
+
+- The secret must be named `license`
+- The secret must be in the same Kubernetes namespace as the Teleport pod
+- The license file in the secret to be named `license.pem`
+
+</TabItem>
+</Tabs>
+
+Only Auth Service instances require the license. Instances of other Teleport
+services do not need a license file unless they are also running the Auth
+Service.
+
+### Check your license expiration date
+
+Your Teleport Enterprise license contains an X.509 certificate that contains the
+expiration date of your license. 
+
+Run the following command to inspect the expiration date of your license,
+assuming your license is saved as `license.pem`:
 
-Only Auth Servers require the license. Proxies and Nodes that do not also run the Auth Service do not need the license.  
-    
-Attempts to use unlicensed products will result in an error message and users will be unable to use those products. For example, if you attempt to use Database Access without it being included in the license, you will get a message similar to:
 ```code
-this Teleport cluster is not licensed for database access, please contact the cluster administrator
+$ openssl x509 -text -in license.pem | grep "Not After"
+             Not After : Dec 16 19:43:40 2022 GMT
+```
+
+### Renew or update your license
+
+If you have changed your license agreement with Teleport, e.g., you have added
+or removed support for a feature, you must obtain an updated license file and
+replace your existing license file on all Teleport Auth Service instances. You
+must also obtain a new license file after renewing your Teleport Enterprise
+license.
+
+Once you restart the Teleport Auth Service, any changes to the license will take
+effect.
+
+The Teleport Auth Service checks the expiration status of your license every
+hour. This means that, after you replace your license file and restart your Auth
+Service instances, it can take up to an hour to confirm the new license. 
+
+## When your license expires
+
+At 90 days before your Teleport Enterprise license expires, users will see an
+alert similar to the following when running `tsh login`, `tsh status`, or any
+`tctl command`:
```

Comment:
@ptgott can you clarifying what dismissing the message will do?

It should disable the message for the duration of the session.